components/opensea/sources/events/events.mjs

```diff
@@ -0,0 +1,55 @@
+import { axios } from "@pipedream/platform";
+import opensea from "../../opensea.app.mjs";
+
+export default {
+  name: "New OpenSea Collection Events",
+  version: "0.0.1",
+  key: "opensea-events",
+  description: "Emit new filtered OpenSea events",
+  props: {
+    opensea,
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 60,
+      },
+    },
+    contractAddress: {
+      type: "string",
+      label: "Contract Address",
+      description: "Collection contract address",
+    },
+    eventType: {
+      type: "string",
+      options: [
+        "sales",
+        "listings",
+      ],
+      label: "Event Type",
+      description: "OpenSea event type",
+    },
+  },
+  dedupe: "greatest",
+  type: "source",
```

Comment:
`type` and `dedupe` attributes may be moved up above `props` for easier visualization